src/xunit.netcore.extensions/Discoverers/ActiveIssueDiscoverer.cs

```diff
@@ -28,17 +28,40 @@ public class ActiveIssueDiscoverer : ITraitDiscoverer
             Debug.Assert(ctorArgs.Count() >= 2);
 
             string issue = ctorArgs.First().ToString();
-            TestPlatforms platforms = (TestPlatforms)ctorArgs.Last();
+            TestPlatforms platforms = TestPlatforms.Any;
+            TargetFrameworkMonikers frameworks = (TargetFrameworkMonikers)~0;
+            
+            foreach (object arg in ctorArgs.Skip(1)) // First argument is the issue number.
+            {
+                if (arg is TestPlatforms)
+                {
+                    platforms = (TestPlatforms)arg;
+                }
+                else
+                {
+                    frameworks = (TargetFrameworkMonikers)arg;
+                }
+            }
+        
             if ((platforms.HasFlag(TestPlatforms.FreeBSD) && RuntimeInformation.IsOSPlatform(OSPlatform.Create("FREEBSD"))) ||
                 (platforms.HasFlag(TestPlatforms.Linux) && RuntimeInformation.IsOSPlatform(OSPlatform.Linux)) ||
                 (platforms.HasFlag(TestPlatforms.NetBSD) && RuntimeInformation.IsOSPlatform(OSPlatform.Create("NETBSD"))) ||
                 (platforms.HasFlag(TestPlatforms.OSX) && RuntimeInformation.IsOSPlatform(OSPlatform.OSX)) ||
                 (platforms.HasFlag(TestPlatforms.Windows) && RuntimeInformation.IsOSPlatform(OSPlatform.Windows)))
             {
-                yield return new KeyValuePair<string, string>(XunitConstants.Category, XunitConstants.Failing);
```

Comment:
So lets say TargetGroup  = netfx and a test has [ActiveIssue(1232)] the default values would be passed, `TestPlatforms.Any` and `(TargetFrameworkMonikers)~0`. Then the first if will be true as platform.HasFlag(TestPlatforms.Windows) will be true, so then since framework = ~0 all the flags are present for it. So it will return:
```
category nonnetcoreapptests
category nonnetfxtests
category nonuaptests
category nonuapaottests
category noncorerttests
```

And in tests.targets since `_bc_TargetGroup = netfx` it will call xunit with argument `-notrait category=nonnetfxtests` so it will skip the test as the discoverer returned a `KeyValuePair`with that category. 

Still moving to 0 as it is more reliable and better to follow the category failing when non targetframework and testplatform are passed.